lib/DockRenderer.vala

```diff
@@ -442,6 +442,9 @@ namespace Plank
 					var draw_value = position_manager.get_draw_value_for_item (item);
 					draw_item (item_cr, item, draw_value, frame_time);
 					draw_item_shadow (shadow_cr, item, draw_value);
+					// draw indicators
+					if (draw_value.show_indicator && item.Indicator != IndicatorState.NONE)
```

Comment:
You should use brackets here, as stated in elementary code style documentation.
Otherwise looks good! 👍🏻